apps/src/dance/songs.js

```diff
@@ -51,14 +51,16 @@ export function fetchSignedCookies() {
 /**
  * Decide which song to select based on the song list and appOptions config.
  * @param songManifest
- * @param config {Object} appOptions config object
+ * @param options {Object} set of options that determine which song to select
```

Comment:
I wonder if it would be useful to understand where the set of options come from (appOptions config.level)?

Reply:
Yeah, similar to above I wanted to abstract this a bit since in Lab2 we don't use appOptions, and not all of these are level properties. I can update this comment to say something like `These are typically derived from the level and/or project sources`?

Reply:
Thanks for updating the comment! Looks good.